### PR TITLE
Added more info on the routing example

### PR DESCRIPTION
**Pull request by vikkio88**
- corrected warning due to [this issue](https://github.com/AvaloniaUI/Avalonia/wiki/Application-lifetimes)
- added a better example on how to disable animation (I had to google quite a bit)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/AvaloniaUI/Documentation?pullRequest=287) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/AvaloniaUI/Documentation?pullRequest=287) before we can accept your contribution.<br/><br/>:x: vikkio88 [sign now](https://cla.dotnetfoundation.org/AvaloniaUI/Documentation?pullRequest=287)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/AvaloniaUI/Documentation?pullRequest=287) it.</sub>

File: guides/deep-dives/reactiveui/routing.md [hunk 1]
```diff
@@ -175,9 +205,7 @@ namespace RoutingExample
     {
         public static void Main(string[] args)
         {
-            BuildAvaloniaApp().Start<MainWindow>(
```
Comment (vikkio88) [inline guides/deep-dives/reactiveui/routing.md:178]: this was giving me a warning and suggesting to switch to the newer `StartWithClassicDesktopLifetime` method

Comment (maxkatz6) [reply guides/deep-dives/reactiveui/routing.md:178]: Thanks for modernizing this page!

Comment (vikkio88) [reply guides/deep-dives/reactiveui/routing.md:178]: you are very welcome, it was hard enough to understand everything without the warnings lol.

**Pull request merged**